docs/components/HeaderMenu.js

```diff
@@ -1,29 +1,27 @@
 // @flow strict
 import { type Node } from 'react';
-import { Box, Link as GestaltLink, Text, Tooltip } from 'gestalt';
-import { useAppContext } from './appContext.js';
-import DarkModeButton from './buttons/DarkModeButton.js';
-import LTRButton from './buttons/LTRButton.js';
+import { Box, CompositeZIndex, Link as GestaltLink, Text, Tooltip } from 'gestalt';
 import trackButtonClick from './buttons/trackButtonClick.js';
 
-export default function HeaderMenu({ isHeader }: {| isHeader?: boolean |}): Node {
-  const { colorScheme, setColorScheme, textDirection, setTextDirection } = useAppContext();
-
-  const onChangeColorScheme = () => setColorScheme(colorScheme === 'light' ? 'dark' : 'light');
-
-  const onTextDirectionChange = () => setTextDirection(textDirection === 'rtl' ? 'ltr' : 'rtl');
-
+export default function HeaderMenu({
+  isHeader,
+  popoverZIndex,
+}: {|
+  isHeader?: boolean,
+  popoverZIndex?: CompositeZIndex,
+|}): Node {
   return (
     <Box
       alignItems="center"
       display={isHeader ? 'none' : 'flex'}
       mdDisplay={isHeader ? 'flex' : 'none'}
       justifyContent={isHeader ? undefined : 'center'}
     >
-      <LTRButton onClick={() => onTextDirectionChange()} textDirection={textDirection} />
-      <DarkModeButton colorScheme={colorScheme} onClick={() => onChangeColorScheme()} />
-
-      <Tooltip inline text="Opens CodeSandbox ready to start coding with Gestalt">
+      <Tooltip
+        inline
+        text="Opens CodeSandbox ready to start coding with Gestalt"
+        zIndex={popoverZIndex}
```

Comment:
Same here and below, this plops the tooltip above the header

Reply:
"plops" is a great verb here 😂